### PR TITLE
Button group use platform touchable 

### LINKED ISSUE #1083: ButtonGroup should use Platform Touchable
Currently, right now the `ButtonGroup` uses `TouchableOpacity` on both platforms. 

Instead, it should use `TouchableOpacity` on iOS and `TouchableNativeFeedback` on Android.

### PR DESCRIPTION
**Pull request by Gregjarvez**
fixes #1083

### CODE REVIEW

**Comment by iRoachie** [comment]
Great PR @Gregjarvez. You just need to do a rebase so you won't have the the `searchBar` commit on this branch

**Comment by Gregjarvez** [comment]
@iRoachie sure will do that. rebase next or master  onto my current working branch ?

**Comment by iRoachie** [comment]
You'll need to rebase from `next`.

`git remote add upstream https://github.com/react-native-training/react-native-elements.git`
`git fetch --all`
`git rebase upstream/next`

**Lastly**
`git push -f origin touchable-specific`

**Comment by Gregjarvez** [comment]
@iRoachie this is awesome. initially when this happened i had to checkout another branch and cherry pick that commit only :).. Thanks !!

**Comment by iRoachie** [comment]
Ha it works pretty sweet. The only downside is that sometimes you get some merge conflicts that you have to go through like if your branch is **wayy** out of sync. Glad I could help.

File: src/buttons/ButtonGroup.js [hunk 1]
```diff
@@ -37,7 +39,10 @@ const ButtonGroup = props => {
     ...attributes
   } = props;
 
-  const Component = component || TouchableHighlight;
+  const Component =
+    component || Platform.OS === 'ios'
```
Comment (iRoachie) [inline src/buttons/ButtonGroup.js]: Lets move this logic to the `defaultProps` section at the bottom

Comment (Gregjarvez) [reply src/buttons/ButtonGroup.js]: sure !!!

**Pull request merged**